definitions.json

```diff
@@ -36,6 +36,20 @@
                         "CommittedTheft"
                     ]
                 },
+                "RelayerEvent": {
```

Comment:
The events _are_ auto-generated, so we only need the new rpc signature if no other types changed.

Reply:
I think the naming is misleading here, this is an internal representation used by the sla pallet.